app/javascript/serviceworker/record-route.js

```diff
@@ -22,7 +22,9 @@ export const recordRouteHandler = async ({ request }) => {
     const campaignId = getCampaignIdFromURL(request.url);
     const campaignUrl = recordTemplateURL(campaignId);
 
-    saveRequest(clonedRequest);
+    const url = clonedRequest.url + "-async";
```

Comment:
End to end tests indirectly check that this is hooked up correctly, but though it's covered by unit tests, we don't assert against it. Should we?